gmapsz/src/main/java/org/zkoss/gmaps/Gmaps.java

```diff
@@ -923,15 +924,40 @@ public String getLibraries () {
 	 * load geometry library only, you can check libraries
 	 * <a href="https://developers.google.com/maps/documentation/javascript/libraries">here</a>
 	 * <p> Use comma to separate different library
-	 * 
+	 *
 	 * @param libraries comma separated String of libraries to load
 	 * @since 3.0.0
 	 */
 	public void setLibraries (String libraries) {
 		setGmapsApiConfigParam(LIBRARIES, libraries);
 	}
 
-	/** 
+	/**
+	 * Returns the last value provided to setExtraMapOptions
+	 * @return unmodifiable Map
+	 * @since 3.1.0
+	 */
+	public Map<String, ?> getExtraMapOptions() {
+		return Collections.unmodifiableMap(_extraMapOptions);
```

Comment:
Calling this without setting _extraMapOptions (`null`) could throw an NPE

Reply:
return emptyMap if null